azurerm/resource_arm_firewall_application_rule_collection.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"log"
 
-	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-06-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-07-01/network"
```

Comment:
since we're bumping the API version here could we do this for all of the networking resources?